addons/hr_timesheet/__manifest__.py

```diff
@@ -1,9 +1,8 @@
 # -*- coding: utf-8 -*-
```

Comment:
You can remove this line from all `__manifest__.py` files you change in both of your PRs :innocent:
Encoding in utf-8 is always done implicitly now :)

Reply:
@deneuvillem  Done with the changes! :100: 
Thanks for the review :innocent: